src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/grazie/GrazieRequestManager.kt

```diff
@@ -51,15 +49,15 @@ class GrazieRequestManager(project: Project) : IJRequestManager(project) {
                         }
                     }
                 }
-            } else {
-                testsAssembler = requestResult.second
             }
-        }
-        catch (e: ClassNotFoundException) {
+//            else {
+//                testsAssembler = requestResult.second
+//            }
```

Comment:
perhaps, it is better to remove it entirely.

Reply:
Addressed by [8e1a82a](https://github.com/JetBrains-Research/TestSpark/commit/8e1a82a0c15b67c955f15ec157cf57d52112d523)